src/test/resources/UserRegistration.feature

```diff
@@ -0,0 +1,47 @@
+Feature: User Registration
+  As a user (/admin)
+  I want to be able to register (users) with username and password
+
+  Background:
+    Given database is empty
+    And user with id 1234 exists and has username "user", password "secure_password" and refreshToken "token"
+
+  Scenario: Successful registration with username, password and password confirmation.
+    When user requests registration with username "kangaroo", password "pig-system" and password confirmation "pig-system"
+    Then response status code is 201
+    And response contains refreshToken "new-token" and the user with id "1235"
```

Comment:
why does the request return the refreshtoken of the newly created user? Should the request return anything at all?